mkt/reviewers/views.py

```diff
@@ -448,17 +444,12 @@ def app_review(request, addon):
         # them to index the app or call update_version() if that wasn't done
         # before already.
         if request.method == 'POST':
-            try:
+            with transaction.atomic():
```

Comment:
(Also, we should file a bug to replace this code, the post_request tasks system we have now make this obsolete)